assets/js/components/PremiumPill/PremiumPill.stories.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PremiumPill from '.';
+
+export default {
+  title: 'PremiumPill',
+  component: PremiumPill,
+};
+
+export function PremiumPillDefault() {
+  return <PremiumPill />;
+}
+
+export function PremiumPillRed() {
+  return <PremiumPill className="bg-red-100" />;
+}
+
+export function PremiumPillYellow() {
+  return <PremiumPill className="bg-yellow-100 text-black" />;
+}
+
+export function PremiumPillWithMarginLeft() {
+  return <PremiumPill className="ml-6" />;
```

Comment:
I would question the relevance of this variation at an atomical level as it seems redundant. I think the scenario is more applicable in a full page layout. @dottorblaster What do you think?

Reply:
I have a similar opinion, but why did I put it in ? I put it in because we use it in two places in the project.
The main question is, should we cover everything how it is used in the project or how you suggest @jagabomb  and leave it out?
@dottorblaster  what do you think ?

Reply:
@EMaksy In this instance it is so minimal and the margin is something that can change dependent on the page application. This why I feel it is stronger if you show a full page with the application of the PremiumPill but once again it is out of scope for this story in my opinion. So I would say leave it out. Let's see what @dottorblaster advises.

Reply:
I see you took your decision without my advice :D anyway yes it was pretty redundant, @jagabomb was right, don't worry @EMaksy!